dnf-behave-tests/dnf/security-advisory.feature

```diff
@@ -55,6 +98,15 @@ Scenario: upgrade-minimal sec-severity
         | Action        | Package                   |
         | upgrade       | advisory_B-0:1.0-2.x86_64 |
 
+Scenario: check-upgrade --minimal with sec-severity
+   When I execute dnf4 with args "upgrade-minimal --sec-severity Moderate"
```

Comment:
I believe the `I execute dnf4` step shouldn't be here, it is not being run, it is a wrong command and these tests are only for dnf5.

Reply:
Thank you - I've replaced dnf4 and dnf5 by simple `dnf`